eternalcore-core/src/main/java/com/eternalcode/core/feature/ignore/UnIgnoreCommand.java

```diff
@@ -47,6 +56,15 @@ void ignore(@Context User sender, @Arg User target) {
                 return;
             }
 
+            Player senderPlayer = this.server.getPlayer(senderUuid);
+            Player targetPlayer = this.server.getPlayer(targetUuid);
+
+            UnIgnoreEvent event = this.eventCaller.callEvent(new UnIgnoreEvent(senderPlayer, targetPlayer));
```

Comment:
Ignore service should call this event.
(Also you don't call ignore purge event)

Reply:
We call every event using the caller, why the case would be different for unignore events?

Reply:
IgnoreServiceImpl doesn't exist, we only have the ORMLite repo which is considered pretty low-level as it only interacts with DB.